rules/linux/credential_access_credential_dumping.toml

```diff
@@ -29,34 +29,37 @@ note = """## Triage and analysis
 ### Investigating Credential Dumping via Unshadow
 
 The rule identifies execution of the unshadow utility which is part of John the Ripper, a password-cracking tool
-on the host machine.
+on the host machine.Malicious actors can use the utility to retrieve the combined contents of the '/etc/shadow' and
+'/etc/password' files.Using the combined file generated from the utility, the malicious threat actors can use them
+as input for password-cracking utilities or prepare themselves for future operations by gathering
+credential information of the victim.
 
 #### Possible investigation steps
 
 - Identify the session entry leader and session user
 - Examine the contents of session leading to the unshadow activity via the Session View.
 - Examine the command execution pattern in the session, which may lead to suspicious activities
 - Examine the command execution post unshadow activity to indicate any exfiltration of shadow file data.
-- Investigate any other alerts or malicious activities associated with the user/host during the past 48 hours.
+- If the analysis has shown exfiltration of cracked password data, analyse the impact of data that has been exfiltrated.
+- Investigate other alerts associated with the user/host during the past 48 hours.
 
 ### False positive analysis
 
-- System administrators trying to retrive passwords.
+- System administrators trying to retrieve passwords.
```

Comment:
I think this can be removed, that should just never happen. I think this rule will not have any FPs unless the query fails. The behavior is just malicious.